react/components/AutocompleteInput/SearchInput/index.tsx

```diff
@@ -95,6 +83,9 @@ const SearchInput: React.FC<PropTypes.InferProps<typeof propTypes> &
           <span
             className="absolute c-muted-3 fw5 flex items-center ph3 t-body top-0 right-0 h-100 pointer"
             onClick={handleClear}
+            role="button"
+            tabIndex={0}
+            onKeyDown={() => null}
```

Comment:
Useful?

Reply:
Yeah. It just solves the a11y warning. Did not want to add new features in this fix/lint PR's